### DISCUSSION TITLE
Configuration of Apollo Client in faustwp/core

### DISCUSSION THREAD

**Discussion by Julhol-droid** [wpengine/faustjs#1227] [Q&A] [answered]
Hello I was wondering how I can configure the Apollo Client in the new Version of faustwp/core.

Unfortunately the docs for this are currently empty.

Thanks in advance.

**Comment by theodesp** [comment, marked as answer]
Hey @Julhol-droid. Thank you for asking. You can definitely configure the Apollo Client using a plugin hook. 
We have a documentation page for that here:

https://faustjs.org/docs/plugin-system/filters#apolloclientoptions

Take a look at an example of how to implement a hook in this blueprint:

https://github.com/wpengine/atlas-blueprint-portfolio/blob/main/plugins/RelayStylePaginationPlugin.js

**Reply by Julhol-droid** [reply]
Thank you very much👍
